projects/plugins/jetpack/extensions/blocks/one-payment/variations.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * The different payment blocks that can be chosen.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { settings as DonationSettings } from '../donations';
+import { settings as RecurringPaymentSettings } from '../recurring-payments';
+
+const variations = [
+	{
+		name: 'onepayment-donations',
+		title: DonationSettings.title,
+		description: DonationSettings.description,
+		icon: DonationSettings.icon.src,
+		innerBlocks: [ [ 'jetpack/donations', {} ] ],
+		// The inner block itself is already listed in the inserter in its own right, so just include in this blocks
+		// unified intro.
+		scope: [ 'block' ],
+	},
+	{
+		name: 'onepayment-recurring-payments',
+		title: RecurringPaymentSettings.title,
+		description: RecurringPaymentSettings.description,
+		icon: RecurringPaymentSettings.icon.src,
```

Comment:
Need to make all icons the same size, ideally programmatically. Currently this one is much smaller:

![image](https://user-images.githubusercontent.com/93301/151216773-6c82d928-f69f-438f-906a-5158aaa0d406.png)